scripts.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+window.addEventListener('load', main);
+document.addEventListener('click', countClick);
+
+const number = document.querySelector('#count');
+
+function main(){
+  if (!sessionStorage.count) {
+    sessionStorage.count = 0;
```

Comment:
sessionStorage doesn't have a method count

Reply:
Its not a method, but a key-value pair. I'm initializing it in function main()